src/tooltip/tooltip.tsx

```diff
@@ -6,7 +6,7 @@ import { TooltipArg, Trigger } from 'react-popper-tooltip/dist/types';
 import PopperJS from 'popper.js';
 
 interface Props {
-    text?: string;
+    text?: any;
```

Comment:
Why would you pass it sth different from a string?

Reply:
yes i need to pass a div

Reply:
What about making it then `string | ReactNode` ? Is it actually supported to do so @MostafaBalata ?

Reply:
Yes, actually one of the reasons why `react-popper-tooltip` because it supports passing react component within the tooltip. I would also make it `string | ReactNode` to be more precisely.